scripts/sensorshow

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+
+"""
+    Script to show Voltage and Current Sensor status.
+"""
+from tabulate import tabulate
+from swsscommon.swsscommon import SonicV2Connector
+from natsort import natsorted
+import argparse
+
+
+header = ['Sensor', 'Voltage(mV)', 'High TH', 'Low TH', 'Crit High TH', 'Crit Low TH', 'Warning', 'Timestamp']
+
+TIMESTAMP_FIELD_NAME = 'timestamp'
+HIGH_THRESH_FIELD_NAME = 'high_threshold'
+LOW_THRESH_FIELD_NAME = 'low_threshold'
+CRIT_HIGH_THRESH_FIELD_NAME = 'critical_high_threshold'
+CRIT_LOW_THRESH_FIELD_NAME = 'critical_low_threshold'
+WARNING_STATUS_FIELD_NAME = 'warning_status'
+
+
+class SensorShow(object):
+    def __init__(self, type):
+        self.db = SonicV2Connector(host="127.0.0.1")
+        self.db.connect(self.db.STATE_DB)
+        self.field_name = type
+        if type == "voltage":
+            self.table_name = 'VOLTAGE_INFO'
+            header[1]  = 'Voltage(mV)'
```

Comment:
This should consume `get_unit`, though it makes me question how `get_unit` may be used; there's no guarantee that all sensors must have the same units. A particular unit should be required for all sensors of any given type, be a constant, and be documented as such.

Reply:
As discussed, will update documentation to mention voltage sensors should report voltages in mV.